ql/src/java/org/apache/hadoop/hive/ql/ddl/table/CreateTableOperation.java

```diff
@@ -140,6 +145,26 @@ private void createTableNonReplaceMode(Table tbl) throws HiveException {
       Table createdTable = context.getDb().getTable(tbl.getDbName(), tbl.getTableName());
       DataContainer dc = new DataContainer(createdTable.getTTable());
       context.getQueryState().getLineageState().setLineage(createdTable.getPath(), dc, createdTable.getCols());
+
+      // We did not create the table when moving the data files for a non-partitioned table and
```

Comment:
update the comment --need to explain why special handling for ctas of transactional table. Its because the move task is created with movefile desc and not load table desc for transactional table ctas.

Reply:
Done. I think it's better to refer to getFileSinkPlan() instead of explaining everything that function does here.